mobilecoind/src/service.rs

```diff
@@ -5977,4 +5988,29 @@ mod test {
             .expect("Failed getting processed block");
         assert_eq!(response.get_tx_outs().len(), 1);
     }
+
+    #[test_with_logger]
+    fn test_get_release_version(logger: Logger) {
+        let mut rng: StdRng = SeedableRng::from_seed([23u8; 32]);
+        let sender = AccountKey::random(&mut rng);
+
+        // 1 known recipient, 3 random recipients and no monitors.
+        let (_ledger_db, _mobilecoind_db, client, _server, _server_conn_manager) =
+            get_testing_environment(
+                BLOCK_VERSION,
+                3,
+                &[sender.default_subaddress()],
+                &[],
+                logger.clone(),
+                &mut rng,
+            );
+
+        let expected_version = String("1.3.0-pre0");
+
+        // Send request.
+        let response = client
+            .get_release_version()
+            .expect("Incorrect version returned");
+        assert_eq!(expected_version, response.get_version());
```

Comment:
It seems more robust to not hardcode a version.

```suggestion
        // Send request.
        let response = client
            .get_release_version()
            .expect("Incorrect version returned");
        assert!(!response.version.is_empty());
```